PaySlip_DelRosario.java

```diff
@@ -54,21 +54,26 @@ public double getGrossIncome() {
     return grossIncome;
   }
 
-  public double getWithholdingTax() {
+  // computes the withholdingTax
```

Comment:
This comment is not needed because you can infer that `getWithholdingTax` is computing the withholdingTax. Comments are normally reserved for `why` is this code is needed (if applicable), but the code itself should explain the `what` and `how`